docs/cookbooks/testing.md

```diff
@@ -10,7 +10,7 @@ python3.8 -m venv venv/
 ./venv/bin/pip install pip-tools -c requirements.in
 
 # install python dependencies with pip-sync (provided by pip-tools)
-./venv/bin/pip-sync
+./venv/bin/pip-sync --pip-args=--no-deps
```

Comment:
Similarly:
```suggestion
./venv/bin/pip-sync --pip-args=--no-deps requirements.txt java-requirements.txt
```

Reply:
yes, this spot should include java requirements, because it runs integration tests below